src/loaders/common/loader_helper.py

```diff
@@ -33,6 +33,29 @@
 """
 This module contains helper functions used for loaders (e.g. compute_genome_attribs, gtdb_genome_attribs_loader, etc.)
 """
+from pydantic import BaseModel
+
+
+def model_to_dict(model_instance) -> dict:
```

Comment:
Can't you just use https://docs.pydantic.dev/1.10/usage/exporting_models/#modeldict

Reply:
https://github.com/kbase/collections/blob/main/src/service/storage_arango.py#L23

is another option and what the arango DAO uses to convert models before DB insertion

Reply:
👍  use `model_dump` instead.

Reply:
:+1: